cmd/test/main.go

```diff
@@ -190,7 +197,18 @@ func New(ctx *TestCommandContext) *cobra.Command {
 				return err
 			}
 
-			testCommandOptions := GenerateTestCommandOptions(testCommandFlags, localConfigContent)
+			prerunDataForEvaluation, statusCode, err := ctx.CliClient.RequestPrerunDataForEvaluation(localConfigContent.CliId)
+
+			// getting prerun data can return 404 if user has a valid token but he didn't sign up yet - suppose to be ok
```

Comment:
you can add this data inside the client, so it will return the results according to the logic written in the comments, that way it will be integrated in the code and not be hidden in the comments, also if we will want to use it in another place we will have to rewrite the same logic in the new place